internal/testing/e2e/consts.go

```diff
@@ -18,32 +18,27 @@ var (
 	zygote OM
 
 	// Indices covers all ticketIndices that will be used in the e2e test
-	// Please update #L394 in the Makefile if you need to add new indices to the e2e tests.
-	// TODO: Unify these indices
-	Indices = []string{SkillAttribute, Map1Attribute, Map2Attribute, LevelIndex, DefenseIndex, BoolIndex, StringIndex}
+	// Please update the ticketIndices in the helm chart for in-cluster end-to-end test if you need to add new indices.
+	Indices = []string{AttributeMMR, AttributeLevel, AttributeDefense, ModeDemo, RoleCleric}
 )
 
 const (
-	// Map1BeginnerPool is an index.
+	// Map1BeginnerPool is a pool name.
 	Map1BeginnerPool = "map1beginner"
-	// Map1AdvancedPool is an index.
+	// Map1AdvancedPool is pool name.
 	Map1AdvancedPool = "map1advanced"
-	// Map2BeginnerPool is an index.
+	// Map2BeginnerPool is pool name.
 	Map2BeginnerPool = "map2beginner"
-	// Map2AdvancedPool is an index.
+	// Map2AdvancedPool is pool name.
 	Map2AdvancedPool = "map2advanced"
-	// SkillAttribute is an index.
-	SkillAttribute = "skill"
-	// Map1Attribute is an index.
-	Map1Attribute = "map1"
-	// Map2Attribute is an index.
-	Map2Attribute = "map2"
-	// LevelIndex is an index
-	LevelIndex = "level"
-	// DefenseIndex is an index
-	DefenseIndex = "defense"
-	// BoolIndex is an index used to test BoolEqualsFilter
-	BoolIndex = "bool"
-	// StringIndex is an index used to test StringEqualsFilter
-	StringIndex = "string"
+	// AttributeMMR is an index used to test FloatRangeFilter.
+	AttributeMMR = "attribute.mmr"
+	// AttributeLevel is an index used to test FloatRangeFilter.
+	AttributeLevel = "attribute.level"
+	// AttributeDefense is an index used to test FloatRangeFilter.
+	AttributeDefense = "attribute.defense"
+	// ModeDemo is an index used to test BoolEqualsFilter
+	ModeDemo = "mode.demo"
+	// RoleCleric is an index used to test StringEqualsFilter
+	RoleCleric = "char.cleric"
```

Comment:
Looks like we are using the string attribute here as a bool by setting its value as "true". Currently, the intent of the PR is to switch to the new fields and remove discrepancy between in cluster / in memory - so i ll unblock that by approval - also, as test case, this does test the different filters to that is ok.

As a realistic example though, you may want to have a attribute "character" and its value as "cleric" or "paladin".. and so forth - rather than "char.cleric" with value true. Please add a TODO for this improvement / create an issue.